app/forms/webauthn_verification_form.rb

```diff
@@ -53,19 +53,20 @@ def valid_assertion_response?(protocol)
       authenticator_data: Base64.decode64(@authenticator_data),
       client_data_json: Base64.decode64(@client_data_json),
       signature: Base64.decode64(@signature),
-      credential_id: Base64.decode64(@credential_id),
+      # Evidently unneeded now: credential_id: Base64.decode64(@credential_id),
```

Comment:
if unneeded, just delete it instead of commenting out?

Reply:
yep, yep totally.  just leaving in there while I got things working